cli/src/main/scala/scalafix/cli/Cli.scala

```diff
@@ -44,6 +45,16 @@ case class ScalafixOptions(
       ".scalafix.conf OR imports.organize=false"
     ) @ExtraName("c") config: Option[ScalafixConfig] = None,
     @HelpMessage(
+      "The classpath for the input files, to support semantic fixes."
+    ) @ValueDescription(
+      "entry1.jar:entry2.jar"
```

Comment:
Can also be a directory like `core/target/classes`, we can reuse the docstring from `Mirror.apply`